moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -57,19 +57,23 @@ namespace rdf_loader
 {
 static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_rdf_loader.rdf_loader");
 
-RDFLoader::RDFLoader(const std::shared_ptr<rclcpp::Node>& node, const std::string& ros_name) : ros_name_(ros_name)
+RDFLoader::RDFLoader(const std::shared_ptr<rclcpp::Node>& node, const std::string& ros_name,
+                     bool default_continuous_value, double default_timeout)
+  : ros_name_(ros_name)
 {
   moveit::tools::Profiler::ScopedStart prof_start;
   moveit::tools::Profiler::ScopedBlock prof_block("RDFLoader(robot_description)");
 
   auto start = node->now();
 
-  urdf_string_ = urdf_ssp_.loadInitialValue(node, ros_name,
-                                            std::bind(&RDFLoader::urdfUpdateCallback, this, std::placeholders::_1));
+  urdf_string_ =
+      urdf_ssp_.loadInitialValue(node, ros_name, std::bind(&RDFLoader::urdfUpdateCallback, this, std::placeholders::_1),
+                                 default_continuous_value, default_timeout);
 
   const std::string srdf_name = ros_name + "_semantic";
   srdf_string_ = srdf_ssp_.loadInitialValue(node, srdf_name,
-                                            std::bind(&RDFLoader::srdfUpdateCallback, this, std::placeholders::_1));
+                                            std::bind(&RDFLoader::srdfUpdateCallback, this, std::placeholders::_1),
```

Comment:
Feel free to disregard this as we don't consistently follow this advice in the codebase but it would be nice to move away from bind if possible.  https://github.com/cpp-best-practices/cppbestpractices/blob/master/08-Considering_Performance.md#never-use-stdbind

```
[&](const auto& new_srdf_string) -> void { srdfUpdateCallback(new_srdf_string); }
```

Reply:
I hate that syntax, so I probably will disregard. (Square brackets to me say array, not lambda function)

Reply:
Ok, that is fair.  I think the reason for the advice is that most people find lambdas easier to read than the bind function and they tend to compile faster than bind.

I honestly don't have a problem with the bind syntax and it is used everywhere in moveit and ros already.  This was just a minor thing I thought I would point out if you would prefer it.